Terminal.Gui/Views/ComboBox.cs

```diff
@@ -108,18 +118,20 @@ private void Initialize()
 			ColorScheme = Colors.Base;
 
 			search.TextChanged += Search_Changed;
+			listview.OpenSelectedItem += (ListViewItemEventArgs a) => Selected();
 
 			// On resize
 			LayoutComplete += (LayoutEventArgs a) => {
 
 				search.Width = Bounds.Width;
 				listview.Width = autoHide ? Bounds.Width - 1 : Bounds.Width;
+				listview.Height = CalculatetHeight ();
 			};
 
 			listview.SelectedItemChanged += (ListViewItemEventArgs e) => {
 
 				if(searchset.Count > 0)
```

Comment:
Will you please use brackets (`{` and `}`) around all nested statements?

Reply:
That's done on my latest check-in